sidecar/src/figwheel_sidecar/repl.clj

```diff
@@ -83,7 +83,10 @@
                        (first (string/split x #"\?")))))
        stack-trace))
 
-(defrecord FigwheelEnv [figwheel-server]
+(defrecord FigwheelEnv [figwheel-server repl-opts]
+  cljs.repl/IReplEnvOptions
```

Comment:
https://github.com/clojure/clojurescript/commit/1e5580c92f4035f8daf941e3c884c9ad4b89c419

First version to include this should be 0.0-2719